# This .patch file removes the openssl-3.0.1.gemspec file from chef-workstation
# for the correct update to chef in redBorder
--- /opt/chef-workstation/embedded/lib/ruby/gems/3.1.0/specifications/default/openssl-3.0.1.gemspec
+++ /dev/null
@@ -1 +0,0 @@
-<file content>
